cocos/ui/UIListView.h

```diff
@@ -244,6 +244,18 @@ class CC_GUI_DLL ListView : public ScrollView
      */
     float getItemsMargin()const;
     
+    /**
+     * Set time to scroll between items (see scrollToItem)
+     * @param time
```

Comment:
I think it is better to mention that, the value should big or equal 0. And please complete the comments. Thanks.